website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -0,0 +1,200 @@
+
+- `Hds::Textarea` fix border and text color for readonly state. [(#1470)](https://github.com/hashicorp/design-system/pull/1470) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `Hds::Modal` prevent onClose callback function invocation when isDismissDisabled is true. [(#1456)](https://github.com/hashicorp/design-system/pull/145) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `font-weight` of the `button` mixin explicitly to `regualr` instead of relying on inheritance (components using this mixin: `Button`, `Dropdown::ToggleButton` and soon `Accordion`) - No visual difference expected. [(#1469)](https://github.com/hashicorp/design-system/pull/1469) Thanks [@didoo](https://github.com/didoo)!
+
+- 🔄 Updated dependencies:
+    - @hashicorp/design-system-tokens@1.6.0
+    - @hashicorp/ember-flight-icons@3.0.7
+
+## 2.7.1
+
+**Month Day, Year**
+
+- `TooltipButton` added `text-align: inherit` to the "button" element. [(#1438)](https://github.com/hashicorp/design-system/pull/1438) Thanks [@didoo](https://github.com/didoo)! 
+
+- Internal accessibility tweaks for dropdown component. [(#1444)](https://github.com/hashicorp/design-system/pull/1444) Thanks [@MelSumner](https://github.com/MelSumner)!
+
+- **Upgraded `Ember.js` to latest stable release 4.12, including upgrades to:**
+
+  - `ember-auto-import` from `2.6.0` to `2.6.3`
+  - `ember-cli-htmlbars` from `6.1.0` to `6.2.0`
+
+- **Upgraded the following dependencies:**
+
+  - `ember-focus-trap` from `1.0.1` to `1.0.2`
+  - `ember-keyboard` from `8.1.0` to `8.2.0`
+  - `ember-truth-helpers` from `3.0.0` to `3.1.1`
+  - `sass` from `1.58.3` to `1.62.1`
+
+- Shifted our supported version of Node.js from `12.* || 14.* || >= 16` to `14.* || 16.* || >= 18`
+
+- [#1395](https://github.com/hashicorp/design-system/pull/1395) Thanks [@alex-ju](https://github.com/alex-ju)! - 
+
+- `Table` set `min-height` instead of `height` for the table head cells + Updated the cells' internal padding to align with the design specs in Figma. [(#1425)](https://github.com/hashicorp/design-system/pull/1425) Thanks [@didoo](https://github.com/didoo)! 
+
+- Updated CSS code of components to use flex `gap`. [(#1433)](https://github.com/hashicorp/design-system/pull/1433) Thanks [@didoo](https://github.com/didoo)! 
+
+- `Hds::Sidenav::Link` fixed an issue that was generating an empty node. This will lead to a minimal visual impact on some edge cases of `Alert/Toast` (multiple description items) and `SideNav` (text + generic content).
+
+- `standalone link` style tweaks. [(#1426)](https://github.com/hashicorp/design-system/pull/1426) Thanks [@MelSumner](https://github.com/MelSumner)! 
+
+- `Hds::Modal` and `Hds::Flyout` fix scroll management resulting in stray `style` attribute on `<body>` elementresulting. [#1434](https://github.com/hashicorp/design-system/pull/1434) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- 🔄 Updated dependencies:
+    - @hashicorp/ember-flight-icons@3.0.6
```

Comment:
```suggestion
**June 26, 2023**

- `TooltipButton` - added `text-align: inherit` to the "button" element. [#1438](https://github.com/hashicorp/design-system/pull/1438) Thanks [@didoo](https://github.com/didoo)! 

- `Dropdown` - added internal accessibility tweaks. [#1444](https://github.com/hashicorp/design-system/pull/1444) Thanks [@MelSumner](https://github.com/MelSumner)!

- `Table` - set `min-height` instead of `height` for the table head cells and updated the cells' internal padding to align with the design specs in Figma. [#1425](https://github.com/hashicorp/design-system/pull/1425) Thanks [@didoo](https://github.com/didoo)! 

- Updated CSS code of components to use flex `gap`.

    - `Hds::Sidenav::Link` - fixed an issue that was generating an empty node. 
    - This will lead to a minimal visual impact on some edge cases of `Alert/Toast` (multiple description items) and `SideNav` (text + generic content).

 [#1433](https://github.com/hashicorp/design-system/pull/1433) Thanks [@didoo](https://github.com/didoo)! 

- Implemented minor style tweaks to: 

    - `Hds::Link::Standalone` - updated to smaller icon size
    - `Hds::Alert` and `Hds::Toast` - updated the space between actions 
    
[#1426](https://github.com/hashicorp/design-system/pull/1426) Thanks [@MelSumner](https://github.com/MelSumner)! 

- `Hds::Modal` and `Hds::Flyout` - fixed scroll management resulting in stray `style` attribute on `<body>` element. [#1434](https://github.com/hashicorp/design-system/pull/1434) Thanks [@alex-ju](https://github.com/alex-ju)!

- Upgraded `Ember.js` to latest stable release 4.12, including upgrades to:

  - `ember-auto-import` from `2.6.0` to `2.6.3`
  - `ember-cli-htmlbars` from `6.1.0` to `6.2.0`

- Upgraded the following dependencies:

  - `ember-focus-trap` from `1.0.1` to `1.0.2`
  - `ember-keyboard` from `8.1.0` to `8.2.0`
  - `ember-truth-helpers` from `3.0.0` to `3.1.1`
  - `sass` from `1.58.3` to `1.62.1`

- Shifted our supported version of Node.js from `12.* || 14.* || >= 16` to `14.* || 16.* || >= 18`

[#1395](https://github.com/hashicorp/design-system/pull/1395) Thanks [@alex-ju](https://github.com/alex-ju)!

- 🔄 Updated dependencies:
    - @hashicorp/ember-flight-icons@3.0.6
```